tests/acceptance/homepage/en-us-test.js

```diff
@@ -1,4 +1,4 @@
-import { visit } from '@ember/test-helpers';
+import { fillIn, find, visit } from '@ember/test-helpers';
```

Comment:
My bad. I realized we didn't need `find`.
```suggestion
import { fillIn, visit } from '@ember/test-helpers';
```